### PR TITLE
Fix README heading with extra hashes

### PR DESCRIPTION
**Pull request by bryant1410**

### CODE REVIEW

**Review by adob — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Then simply
 ```
 at the top of the file, but after the system `#includes`s.
 
-#### Misc####
```
Comment (ilve2be4networks) [inline README.md]: Opt-Out-All-Completely

Comment (ilve2be4networks) [reply README.md]: Opt-Out-All-Completely

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ Then simply
...
+#### Misc ####
```
Comment (ilve2be4networks) [inline README.md]: Opt-Out-All-Completely

File: README.md (hunk 1 above)
Comment (ilve2be4networks) [reply README.md]: Opt-Out-All-Completely

File: README.md (hunk 2 above)
Comment (ilve2be4networks) [reply README.md]: Opt-Out-All-Completely

Comment (ilve2be4networks) [reply README.md]: Opt-Out-All-Completely

File: README.md (hunk 1 above)
Comment (ilve2be4networks) [reply README.md]: Opt-Out-All-Completely-Now

**Review by ilve2be4networks — COMMENTED** [review]
Opt-Out-All-Completely-DO-NOT-TRACK-ME-OR-MINE

**Comment by ilve2be4networks** [comment]
Opt-Out-Merges

**Pull request merged**